ballerina/http_client_connection_pool.bal

```diff
@@ -29,14 +29,22 @@ configurable int maxActiveStreamsPerConnection = 100;
 # + maxActiveStreamsPerConnection - Maximum active streams per connection. This only applies to HTTP/2. Default value is 100
 # + minEvictableIdleTime - Minimum evictable time for an idle connection in seconds. Default value is 5 minutes
 # + timeBetweenEvictionRuns - Time between eviction runs in seconds. Default value is 30 seconds
+# + minIdleTimeInStaleState - Minimum time in seconds for a connection to be kept open which has received a GOAWAY.
+#                             This only applies for HTTP/2. Default value is 5 minutes. If the value is set to -1,
+#                             the connection will not be closed until all existing streams are completed
+# + timeBetweenStaleCheck - Time between the connection stale check run in seconds. This only applies for HTTP/2.
```

Comment:
```suggestion
# + timeBetweenStaleEviction - Time between the connection stale eviction runs in seconds. This only applies for HTTP/2.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-http/pull/1833/commits/feeca1fbb6a4e9b33ff2074d822c532a837c897c

Reply:
Shall we rename the configuration to `timeBetweenStaleEviction`? Since the stale condition is imposed by the GO_AWAY frame, the name `timeBetweenStaleCheck` can be misunderstandable.

Reply:
Updated